ddtrace/internal/telemetry/data/metrics.py

```diff
@@ -0,0 +1,62 @@
+import time
+from typing import Dict
+from typing import List
+from typing import Literal
+from typing import Optional
+from typing import Tuple
+
+from ...hostname import get_hostname
+
+
+class MetricType:
+    """gauge, count, and rate are the 3 metric types accepted by the Telemetry Instrumentation"""
+
+    COUNT = "count"  # type: Literal['count']
+    GAUGE = "gauge"  # type: Literal['gauge']
+    RATE = "rate"  # type: Literal['rate']
+
+
+class Series:
```

Comment:
We do if we want to submit metrics to the telemetry org. Using normal metrics/dogstatsd will submit metrics to the customers org. To get around this we would need to hardcode the API KEY for the telemetry org in our tracers and that would lead to some issues